front/src/assets/rollingStock/freightRollingStocks.ts

```diff
@@ -108,6 +108,7 @@ const FREIGHT_ROLLING_STOCKS = [
   'Y8000AP',
   'Y9000US',
   'electric_rolling_stock_test_e2e',
+  'FAST_ROLLING_STOCK_TEST_E2E',
```

Comment:
Can't we rename the fast rolling stock (used in the e2e test) to one of the whitelisted ?

Reply:
At least we can add a TODO comment to explain why we have this value and how we should get rid of this.